### DISCUSSION TITLE
Welcome to echo-speaks Discussions!

### DISCUSSION THREAD

**Discussion by tonesto7** [tonesto7/echo-speaks#430] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tonyshor** [comment]
I am a new Hubitat user and having problems deploying EchoSpeaks app. I keep getting " 404 we couldn't find that page" when attempting to goto login page when trying to link Amazon and Heroku

![image](https://github.com/tonesto7/echo-speaks/assets/34520425/edaa0aeb-84c1-408a-9ea0-9ab9ea9e2a1d)

![image](https://github.com/tonesto7/echo-speaks/assets/34520425/088fcc44-9ee0-41b7-852e-1931caab98b9)

I was able to deploy app in HEROKU successfully

![image](https://github.com/tonesto7/echo-speaks/assets/34520425/2ad14852-687b-49f5-a246-be548a622338)


I deleted the app and re-added multiple times with no success. Not sure what i am doing wrong, Any suggestion? thanks

**Comment by medic563** [comment]
Same problem with the Amazon 404 page. Is this App dead now?
